### DISCUSSION TITLE
Issues when stopping

### DISCUSSION THREAD

**Discussion by entradainteractive** [Sixze/ALS-Refactored#421] [Q&A]
I updated to your latest version, but now there is something wrong when the player stops. I'm sure it's my fault, so I'm looking for any ideas of where to start looking. Thanks!

Here's a short video showing the issue.

https://github.com/Sixze/ALS-Refactored/assets/13300783/d37c1d60-6292-4b8c-a6ea-c64c201737cc

**Comment by ameaninglessname** [comment]
You could use [rewind debugger](https://docs.unrealengine.com/latest/en-US/animation-rewind-debugger-in-unreal-engine/) to check what happened in that frames
